package.json

```diff
@@ -24,14 +24,17 @@
   },
   "homepage": "https://github.com/anna-kuzii/sport-news#readme",
   "dependencies": {
+    "bcryptjs": "^2.4.3",
     "body-parser": "^1.19.0",
     "bootstrap": "^4.3.1",
     "dotenv": "^8.0.0",
     "express": "^4.17.1",
+    "express-validator": "^6.1.1",
     "mongodb": "^3.3.0",
     "mongoose": "^5.6.9",
+    "nodemon": "^1.19.1",
+    "passport": "^0.4.0"
```

Comment:
add comma and we will not see red highlights

Reply:
add